buildspec.yaml

```diff
@@ -10,10 +10,10 @@ phases:
       - yarn install --frozen-lockfile
   build:
     commands:
-      - /bin/bash ./build.sh
+      - yarn build
```

Comment:
I believe that this loses the [git secret scan](https://github.com/aws/aws-rfdk/blob/a875bc0804c74479e717aa3846c5d290b2ec416a/build.sh#L41C1-L41C32) that we have in the `build.sh`

Reply:
`yarn build` is an alias for `yarn run build` ([ref](https://classic.yarnpkg.com/lang/en/docs/cli/#toc-user-defined-scripts)). And our script just runs `build.sh`:

https://github.com/aws/aws-rfdk/blob/a875bc0804c74479e717aa3846c5d290b2ec416a/package.json#L17

The reason this change was made is because if we run the script directly, we don't get any of the environment setup that yarn does when running scripts.